spec/velveteen/worker_spec.rb

```diff
@@ -64,6 +64,20 @@
       _, properties, _ = queue.pop
       expect(properties[:headers]).to eq("foo" => "bar")
     end
+
+    it "raises an error when published message does not match routing key schema" do
+      message = Velveteen::Message.new(data: {"foo" => "bar"}, headers: {})
+      worker = TestPublishingWorker.new(message: message)
+      queue = Velveteen::Config.channel.queue("")
```

Comment:
If you aren't concerned with the queue name, you can omit the parameter (`""` is the default value).

Reply:
Great idea